src/frontends/tensorflow/src/op/interpolate.cpp

```diff
@@ -13,34 +13,43 @@ namespace frontend {
 namespace tensorflow {
 namespace op {
 ov::OutputVector translate_interpolate_op(const NodeContext& node) {
-    auto input = node.get_input(0);
-    auto input_sizes = node.get_input(1);
+    default_op_checks(node, 2, {"ResizeBilinear", "ResizeNearestNeighbor"});
+    auto images = node.get_input(0);
+    auto size = node.get_input(1);
 
+    // retrieve optional attribute
+    auto tf_align_corners = node.get_attribute<bool>("align_corners", false);
+
+    // prepare attributes for OpenVINO Interpolate operation
     Interpolate::InterpolateAttrs interpolate_attrs;
     interpolate_attrs.mode = Interpolate::InterpolateMode::LINEAR;
     interpolate_attrs.shape_calculation_mode = Interpolate::ShapeCalcMode::SIZES;
-    if (node.get_attribute<bool>("align_corners", false))
+    if (tf_align_corners) {
```

Comment:
Those operations also have mode `half_pixel_centers` can we add support for it as well? Or at least complain if we meet `half_pixel_centers=true`

Reply:
Great catch! I am done to align TF FE with legacy frontend. Now it considers different cases.